modules/client-common/src/context-core.ts

```diff
@@ -22,7 +22,7 @@ import {
 } from "./errors";
 
 const DEFAULT_GAS_FEE_ESTIMATION_FACTOR = 0.625;
-const supportedProtocols = ["https:"];
+const supportedProtocols = ["https:", "http:"];
```

Comment:
If we add ths i would delete the next lines because they wouldnt makes sense anymore